api/api/spec/spec.yaml

```diff
@@ -29,6 +33,11 @@ info:
 
     `curl -k -X <METHOD> "https://<HOST_IP>:55000/<ENDPOINT>" -H  "Authorization: Bearer <YOUR_JWT_TOKEN>"`
 
+    Change the token base duration:
+
+    `curl -k -X PUT "https://<HOST_IP>:55000/security/config" -H "Authorization: Bearer <YOUR_JWT_TOKEN>"
+    -d '{"auth_token_exp_timeout":"<NEW_EXPIRE_TIME_IN_SECONDS>"}'`
```

Comment:
Change `"<NEW_EXPIRE_TIME_IN_SECONDS>"` to `<NEW_EXPIRE_TIME_IN_SECONDS>` as `auth_token_exp_timeout` must be of type integer.